webhook/client_test.go

```diff
@@ -212,6 +212,12 @@ func TestConstructEvent_SuccessOnExpectedAPIVersion(t *testing.T) {
 }
 
 func TestConstructEvent_SuccessOnNewAPIVersionInExpectedReleaseTrain(t *testing.T) {
+	// this test only makes sense on GA versions- the exact version for preview versions doesn't
+	// work this way and we can't mock private methods from this test class.
+	if strings.HasSuffix(stripe.APIVersion, ".preview") {
```

Comment:
this was the only manual change- everything else was the result of the merge script (or running unit tests locally)